src/components/film/FilmPage.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import {
+  Container,
+  Darken,
+  Footer,
+  Header,
+  Logo,
+  Row,
+  Spinner,
+} from '../layout/index'
+import { Button } from '../controls/index'
+import MovieDetails from './MovieDetails'
+import ResultsBar from '../search/ResultsBar'
+import SearchResult from '../search/SearchResult'
+import { search, SEARCH_TYPE } from '../../services/searchService'
+import NoResult from '../search/NoResults'
+
+const NavLink = Button.withComponent('a').extend`
+    float: right;
+    margin-top: -8px;
+    text-decoration: none;
+`
+
+class FilmPage extends Component {
+  constructor() {
+    super()
```

Comment:
A good practice is to pass `props` from `constructor` arguments to base class through `super`